ga4gh/datamodel/variants.py

```diff
@@ -509,14 +517,17 @@ def buildMetadata(
             metadata = protocol.VariantSetMetadata()
             metadata.key = key
             metadata.value = value
-            metadata.id = id
             metadata.type = type
```

Comment:
I know you didn't add this line, but change `type` to `type_` while you're at it